Library/Homebrew/formula_creator.rb

```diff
@@ -137,7 +137,7 @@ def install
                                   "--disable-silent-rules",
                                   "--prefix=\#{prefix}"
         <% elsif mode == :go %>
-            system "go", "build", "-o", "\#{bin}/\#{name}"
+            system "go", "build", "-o", bin/name, *std_go_args
```

Comment:
maybe even `-o bin/name` could be part of `std_go_args`? What if you specify multiple `-o`?

Reply:
Only the latter option specified is used.

I updated this PR and the other one accordingly.